File: gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/response/type/OnCreated.kt
package gaia.sdk.response.type

import gaia.sdk.api.scalar.*
import gaia.sdk.api.transmitter.*
import gaia.sdk.response.intf.*
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonCreator
import gaia.sdk.request.enumeration.*

data class OnCreated @JsonCreator constructor(
    @JsonProperty("id") val id:Uuid?, 
    @JsonProperty("identityId") val identityId:Uuid?, 
    @JsonProperty("reference") val reference:Uuid?, 
    @JsonProperty("type") val type:String?
)

File: gaia-sdk-javascript/src/graphql/response/type/OnCreated.ts


import {Uuid, Timestamp, Struct, Long} from "../../GaiaClient";
import {RuntimeState} from "../../request/enumeration/RuntimeState";
import {SkillState} from "../../request/enumeration/SkillState";

export interface OnCreated {
    id?:Uuid, 
    identityId?:Uuid, 
    reference?:Uuid, 
    type?:String
}